data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_equip.lst

```diff
@@ -4,5 +4,8 @@ SOURCELONG:Monster Compendium: Monsters of Faerun	SOURCESHORT:mon	SOURCEDATE:200
 # Equipment Name	Required Proficiency			Type									Cost		Weight	AC Penalty Check	Maximum DEX Bonus	Source Page		% of Spell Failure	Combat bonus
 #Light Armor
 # Equipment Name	Required Armor Proficiency		Type						Cost		Weight	AC Penalty Check	Modifier	Maximum DEX Bonus	Source Page		% of Spell Failure	Combat bonus
-Silverweave Armor	PROFICIENCY:ARMOR|Silverweave Armor	TYPE:Armor.Light.ArmorProfLight.ArmorType_Light.Suit.Silverweave	COST:50	WT:20		ACCHECK:-1		EQMOD:STEEL	MAXDEX:5		SOURCEPAGE:p.	SPELLFAILURE:15		BONUS:COMBAT|AC|3|TYPE=Armor
-Pearl Armor		PROFICIENCY:ARMOR|Pearl Armor		TYPE:Armor.Medium.ArmorProfMedium.ArmorType_Medium.Suit.Silverweave	COST:300	WT:40		ACCHECK:-5		EQMOD:STEEL	MAXDEX:2		SOURCEPAGE:p.	SPELLFAILURE:30		BONUS:COMBAT|AC|5|TYPE=Armor
+Silverweave Armor	PROFICIENCY:ARMOR|Chainmail		TYPE:Armor.Light.ArmorProfLight.ArmorType_Light.Suit.Silverweave	COST:50	WT:20		ACCHECK:-1		EQMOD:STEEL	MAXDEX:5		SOURCEPAGE:p.	SPELLFAILURE:15		BONUS:COMBAT|AC|3|TYPE=Armor
+#Pearl Armor		PROFICIENCY:ARMOR|Pearl Armor		TYPE:Armor.Medium.ArmorProfMedium.ArmorType_Medium.Suit.Silverweave	COST:300	WT:40		ACCHECK:-5		EQMOD:STEEL	MAXDEX:2		SOURCEPAGE:p.	SPELLFAILURE:30		BONUS:COMBAT|AC|5|TYPE=Armor
+#Axe, Throwing			KEY:Axe (Throwing)			PROFICIENCY:WEAPON|Axe (Throwing)			TYPE:Weapon.Container.Melee.Martial Ranged.Finesseable.Ranged.Thrown.Martial.Standard.Slashing.Axe								CONTAINS:UNLIM|Weapon Crystal=1											COST:8	WT:2							CRITMULT:x2	CRITRANGE:1							DAMAGE:1d6				EQMOD:STEEL						WIELD:Light						RANGE:10						SIZE:M	SOURCEPAGE:p.114																																																			SPROP:A throwing axe is lighter than a handaxe and balanced for throwing. Gnome fighters often use throwing axes for both melee and ranged attacks.
+#Gouger Tongue	KEY:Gouger Tongue								TYPE:Natural.Piercing.Finessable.Slashing.Simple		CRITMULT:x2	CRITRANGE:1							DAMAGE:1d8	SIZE:S
```

Comment:
Adding commented out items does not make sense to me. Best to make the correction and remove the old item. In git, we can easily track changes. Also decreases the amount of items to review. Currently you have three Pearl Armor changes that I see - the removed, the commented out, and the new. Only having to compare old and new would be better.